hedera-node/hedera-mono-service/src/main/java/com/hedera/services/contracts/execution/HederaMessageCallProcessor.java

```diff
@@ -87,6 +96,7 @@ public void start(final MessageFrame frame, final OperationTracer operationTrace
                     frame.setExceptionalHaltReason(ILLEGAL_STATE_CHANGE);
                     frame.setState(MessageFrame.State.EXCEPTIONAL_HALT);
                 } else if (updater.get(frame.getRecipientAddress()) == null) {
+                    // can be reached only when EVM_VERSION >= 0.32
```

Comment:
That assertion needs some exposition as to why it's true.

Reply:
Added more details to the comment